src/booleans.js

```diff
@@ -1,62 +1,98 @@
-function negate(a) {
-  // your code here
-};
+const negate = (a) => !a;
 
-function both(a, b) {
-  // your code here
-};
+const both = (a, b) => {
+  if (!a || !b) {
+    return false;
+  }
+  return true;
+}
 
-function either(a, b) {
-  // your code here
-};
+const either = (a, b) => {
+  if (a || b) {
+    return true;
+  }
+  return false;
+}
+const none = (a, b) => {
+  if (a || b) {
+    return false;
+  }
+  return true;
+}
 
-function none(a, b) {
-  // your code here
-};
+const one = (a, b) => {
+  if ((a && !b) || (b && !a)) {
+    return true;
```

Comment:
You can use `return Boolean(((a && !b) || (b && !a)))` or `return !!((a && !b) || (b && !a))` here to get rid of the if/else statement.